node.js

```diff
@@ -663,160 +869,239 @@ async function stabilize() {
             // no need for handler
         }
     }
+
+    /* TBD 20191103 */
+    // update successor table - deviates from SIGCOMM
+    try {
+        await update_successor_table();
+    } catch (err) {
+        // probably no need for error handler
+    }
+    /* TBD 20191103 */
```

Comment:
As mentioned in a previous comment, these sorts of timestamp comments seem to extra noise in the commit. I can get this data from git on a line-by-line basis via Code Lens